testing/database_tests.js

```diff
@@ -0,0 +1,43 @@
+const tape = require('tape');
+const { buildDatabase, makeEmptyTables } = require('../server/database/build.js');
+const getData = require('../server/queries/getData.js');
+const { db } = require('../server/database/dbconnection');
+
+tape('--------------database_tests.js----------check tape is working', (t) => {
+  t.ok(true, 'tape is working');
+  t.equal(process.env.NODE_ENV, 'test', 'The process.env.NODE_ENV environment should be test');
+  t.end();
+});
+
+tape('testing runDbBuild', (t) => {
+  buildDatabase()
```

Comment:
it's much better for you and for you testing environment sanity to build the db once and then just test, i would say consult with @shiryz on that

Reply:
I've had a look at this but will leave it for now.. it will be in the next pull request.

Reply:
did you ask @shiryz

Reply:
Yes. I've spoken to her. I understand how to do it.. will do it this eve.